pandas/compat/__init__.py

```diff
@@ -12,9 +12,16 @@
 import os
 import platform
 import sys
-from typing import TYPE_CHECKING
 
 from pandas._typing import F
+import pandas.compat._compressors
+from pandas.compat._constants import (  # noqa: F401
+    IS64,
+    PY39,
+    PY310,
+    PY311,
+    PYPY,
```

Comment:
Can you use the redundant export `IS64 as IS64`, then type checkers (and linters) know that these imports are meant to be re-exported

Reply:
Yep updated to do that instead. Thanks!

Reply:
Unfortunately `flake8` saw through this, but did an identity assignment, which seemed to fix it while still being explicit about imports. Please let me know what you think.

Reply:
Sorry, I mixed the policy of pandas-stubs and pandas up: pandas-stubs uses the redundant imports in pyi files; pandas re-exports imports by listing all impotable symbols in `__all__` (which is flake8-compatible)

At the bottom of the file, I would add `__all__ = ["PY39", ...]` unfortunately, that means you will also need to list other functions/classes defined in this file (not just those constants).

Reply:
Gotcha. Pushed that change in a recent commit.